packages/fiori/src/ShellBar.hbs

```diff
@@ -8,14 +8,27 @@
 				<slot name="startButton"></slot>
 		{{/if}}
 
-		{{#unless interactiveLogo}}
-			<img class="ui5-shellbar-logo" src="{{logo}}" @click="{{_logoPress}}" />
-		{{/unless}}
+		{{#if hasNonInteractiveLogo}}
+			<img class="ui5-shellbar-logo"
+				src="{{logo}}"
+				@click="{{_logoPress}}"
+				aria-label="Logo"
+				title="Logo"
```

Comment:
-should have a role=button if interactive and also the aria-label and the title
-should be accessible also with keyboard

Reply:
Done